libs/shared/src/consts/handlebar-helpers/getTemplateVariables.ts

```diff
@@ -0,0 +1,91 @@
+import { TemplateVariableTypeEnum } from '../../entities/message-template';
+import { HandlebarHelpersEnum } from './handlebarHelpers';
+
+export interface IMustacheVariable {
+  type: TemplateVariableTypeEnum;
+  name: string;
+  defaultValue?: string | boolean;
+  required?: boolean;
+}
+
+export function getTemplateVariables(bod: any[]): IMustacheVariable[] {
+  const stringVariables: IMustacheVariable[] = bod
+    .filter((body) => body.type === 'MustacheStatement')
+    .flatMap((body) => {
+      const varName = body.params[0]?.original || (body.path.original as string);
+      if (!shouldAddVariable(varName)) {
+        return [];
+      }
+
+      if (body.params[0]?.original) {
+        if (!(Object.values(HandlebarHelpersEnum) as string[]).includes(body.path.original)) {
```

Comment:
This is to handle the cases of handlebar helpers being recognized as vars in the var manager but the actual var being ignored.
For example - {{tiltlecase message}} would have presented titlecase as the var in our var manager + would not show message which is the actual var.

Now will show message as var.
I will make it more clear code-wise but I have a few more debates about how we would like to handle edge cases like {{titlecase var1 var2}} => should it not be recognized at all as a valid var? I think so. But helpers have different numbers of params - like dateFormat or pluralize. I don't want to make it too strict🤔